src/com/homepractice/AbstractionPractice.java

```diff
@@ -0,0 +1,35 @@
+package com.homepractice;
+
+// You can not create object of abstract class..so what is the need of abstract class constructor
+// if abstract class contains 50 fields and also has 100 child class then without abstract class constructor 
+//you have to write initialization in every child class ..reduces the code reusablity
+public abstract class AbstractionPractice {
+
+	 String name;
+	 int age;
+	
+	public AbstractionPractice(String name, int age) {
+		super();
+		this.name = name;
+		this.age = age;
+	}
+	public static void main(String[] args) {
+		
+		
```

Comment:
Can program be run without this main in abstract class?

Reply:
To run program JVM always searches for main method in class....somewhere else we need to write.